assets/src/edit-story/utils/getMediaBaseColor.js

```diff
@@ -65,7 +62,12 @@ function getDefaultOnloadCallback(nodeKey, resolve, reject) {
   return () => {
     try {
       const node = document.body[nodeKey];
-      resolve(thief.getColor(node.firstElementChild));
+      import(
+        /* webpackPrefetch: true, webpackChunkName: "chunk-colorthief" */ 'colorthief'
+      ).then(({ default: ColorThief }) => {
+        const thief = new ColorThief();
+        resolve(thief.getColor(node.firstElementChild));
+      });
     } catch (err) {
       trackError('image_base_color', err.message);
```

Comment:
Does this catch actually catch anything now or is it superfluous?

Reply:
Hm, we can add `catch` directly to the import now, instead.